Tasks/emailReportTask/config/ConfigurationProvider.ts

```diff
@@ -89,6 +89,10 @@ export class ConfigurationProvider implements IConfigurationProvider {
       throw new InputError("Email subject not set");
     }
 
+      // From Address
+      const fromAddress = tl.getInput(TaskConstants.FROMADDRESS_INPUTKEY, true);
+
```

Comment:
Can you remove these blank duplicated break lines?

Reply:
@fahadash

Reply:
Can someone make this go forward please ? Even if there is two additional spaces.

Reply:
@fahadash

Reply:
@fahadash

Reply:
@Naweap Removed the extra whitespaces

Reply:
Now everything seems ok. Can we go forward ? @JaderCM  @fahadash

Reply:
@microsoftopensource can you approve please?

Reply:
Can someone approve this PR so it goes further ? Thanks in advance

Reply:
@microsoftopensource @msftgits @fahadash @msftdata 
Dear @Santa,
Please make this PR to become resolved in a New Year! 
Best wishes from Dev Team!